changelog/0.34.md

```diff
@@ -99,3 +99,18 @@ Released 2023-11-21
 - [#1872](https://github.com/elastisys/compliantkubernetes-apps/pull/1872) - bug: config: fix k8susercrds constraint permission issues for cluster admin @anders-elastisys
 - [#1712](https://github.com/elastisys/compliantkubernetes-apps/pull/1712) - apps sc: thanos receiver out-of-order bug fix @linus-astrom
 - [#1877](https://github.com/elastisys/compliantkubernetes-apps/pull/1877) - apps: Increased validatingWebhookTimeout to 30 seconds and tuned gatekeeper controller manager resources
+
+## v0.34.1
+
+Released 2023-12-19
+<!-- -->
+> [!IMPORTANT]
+> **Platform Administrator Notice(s)**
+> - Fixes issue where log-manager compaction script isn't removing all files as it should
+
+## Changes by kind
+
+### Other(s)
```

Comment:
Also shouldn't these be under kind "Bug(s)" instead of "Other(s)"?

Reply:
Yeah !! I can move these to bugs

Reply:
I see that everything other than features and improvements are put under others by the release script.